pkg/kubelet/dockertools/manager.go

```diff
@@ -1889,3 +1891,12 @@ func (dm *DockerManager) doBackOff(pod *api.Pod, container *api.Container, podSt
 	dm.clearReasonCache(pod, container)
 	return false
 }
+
+// getPidMode returns the pid mode to use on the docker container based on pod.Spec.HostPID.
+func getPidMode(pod *api.Pod) string {
+	pidMode := ""
+	if pod.Spec.HostPID {
+		pidMode = "host"
```

Comment:
@smarterclayton if we can get #12823 in I woukd really like this to go in
afterward.

On Monday, September 14, 2015, Paul Weil <notifications@github.com> wrote:

> In pkg/kubelet/dockertools/manager.go
> <https://github.com/kubernetes/kubernetes/pull/13447#discussion_r39417286>
> :
>
> > @@ -1889,3 +1891,12 @@ func (dm *DockerManager) doBackOff(pod *api.Pod, container *api.Container, podSt
> >  	dm.clearReasonCache(pod, container)
> >  	return false
> >  }
> > +
> > +// getPidMode returns the pid mode to use on the docker container based on pod.Spec.HostPID.
> > +func getPidMode(pod *api.Pod) string {
> > +	pidMode := ""
> > +	if pod.Spec.HostPID {
> > +		pidMode = "host"
>
> I did not see a constant for this in the docker client packages. UTS mode
> and host network also are using "host" in manager.go so I went with that
> approach.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/13447/files#r39417286>.
>